src/BrowserFixes.js

```diff
@@ -0,0 +1,47 @@
+define(function(require) {
+    'use strict';
+
+    var BrowserInfo = require('wf-js-common/BrowserInfo');
+    var DeviceInfo = require('wf-js-common/DeviceInfo');
+
+    var BrowserFixes = function() {
+        // No setup required at this time
+    };
+
+    BrowserFixes.prototype = {
+
+        /**
+         * This applies a workaround for disappearing DOM elements over canvas
+         * content in webkit browsers. The following discussions indicate this
+         * translateZ setting may help.
+         * http://stackoverflow.com/questions/19172249
+         * http://stackoverflow.com/questions/7808110
+         *
+         * @param {HTMLElement} domElement -- the DOM element to apply the fix to
+         */
+        applyDisappearingElementFix: function(domElement) {
+            if (DeviceInfo.browser.webkit && BrowserInfo.hasCssTransforms3d) {
+                domElement.style[BrowserInfo.cssTransformProperty] = 'translateZ(0px)';
+            }
+        },
+
+        /**
+         * Unblit is a companion to `applyDisappearingElementFix`, used to force
+         * the browser to remove aliasing after scaling the target.
+         *
+         * @param {HTMLElement} domElement
+         */
+        unblit: function(domElement) {
```

Comment:
I'm just moving code around here, not looking at the content of it necessarily. ;)